mantidproject/mantid
# Instrument view mask bins

Fixes #13926.

**To test**
1. Open a workspace in the Instrument view.
2. Go to Draw tab.
3. Reduce the integration range using the slider at the bottom of the interface.
4. Draw a shape and "Apply bin mask to View"
5. Apply to data
6. Check that the bins were masked.
7. Check that the detector masking still works. (Restore integration range to full)
8. Check that detector masks can be combined with bin masks in the view and can be applied together to data.
9. Check that all Save options work and labelled correctly.

Wiki page updated:
http://www.mantidproject.org/MantidPlot:_Instrument_View#Changes_coming_in_version_3.6
